### PR TITLE
Release Notes Template PMM

### PR DESCRIPTION
**Pull request by rasika-chivate**
Create Release Notes Template for PMM

### CODE REVIEW

**Review by dygksy — APPROVED** [review]
Thank you Rasika, great template!

**Comment by nastena1606** [comment]
Should the template be the part of the user-facing documentation?

File: docs/release-notes/index.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Release Notes
+- [Percona Monitoring and Management RN Template](Release_notes_Template_PMM.md)
```
Comment (denisok) [inline docs/release-notes/index.md:2]: this will be showed in prod documentation, do we want it ?

File: mkdocs-base.yml [hunk 2]
```diff
@@ -247,6 +247,7 @@ nav:
   - faq.md
   - Release Notes:
       - release-notes/index.md
+      - "PMM RN Template": release-notes/Release_notes_Template_PMM.md
```
Comment (denisok) [inline mkdocs-base.yml:250]: this will be showed in prod documentation, do we want it ?

**Comment by rasika-chivate** [comment]
Created Another PR for this.

**Pull request closed without merging**